File: examples/typescript/bot.js
import { Client, GatewayIntentBits } from 'discord.js';

// Create a new client instance
const client = new Client({ intents: [GatewayIntentBits.Guilds, GatewayIntentBits.GuildMessages] });

// Event: Bot is ready
client.once('ready', () => {
    console.log(`Logged in as ${client.user?.tag}`);
});

// Event: Message received
client.on('messageCreate', (message) => {
    if (message.content === '!ping') {
        message.channel.send('Pong!');
    }
});

// Login to Discord with your bot's token
client.login('YOUR_TOKEN_HERE');

File: examples/python/bot.py
import discord
from discord.ext import commands

# Create a bot instance
bot = commands.Bot(command_prefix="!")

# Event: Bot is ready
@bot.event
async def on_ready():
    print(f"Logged in as {bot.user}")

# Command: Respond to !ping
@bot.command()
async def ping(ctx):
    await ctx.send("Pong!")

# Run the bot
bot.run("YOUR_TOKEN_HERE")
